bundles/specmate-persistency-cdo/src/com/specmate/persistency/validation/NameValidator.java

```diff
@@ -0,0 +1,64 @@
+package com.specmate.persistency.validation;
+
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.emf.ecore.EStructuralFeature;
+
+import com.specmate.common.SpecmateValidationException;
+import com.specmate.model.base.BasePackage;
+import com.specmate.model.base.INamed;
+import com.specmate.persistency.IChangeListener;
+import com.specmate.persistency.event.EChangeKind;
+
+public class NameValidator extends BaseValidator implements IChangeListener {
+	/** Pattern that describes invalid object names */
+	private static Pattern inValidNameChars = Pattern.compile("[,;|]");
```

Comment:
Why are these symbols not valid for names?

Reply:
We use these in transaction commits for field and data separators. I added a comment for clarification.